File: Sources/TLLoggingUtil.m
//
//  LoggingUtil.c
//
//  Created by Thanh Vu on 25/11/2020.
//  Copyright © 2020 Solar. All rights reserved.
//

#include "TLLoggingUtil.h"

void Objc_DCHECK(bool value, NSString *file, NSString *func, int line) {
    DEBUGCHECK(value, file, func, line);
}

File: Sources/TLLoggingUtil.swift
//
//  TLLoggingUtil.swift
//  TrumpVoiceChange
//
//  Created by Thanh Vu on 18/11/2020.
//

import Foundation

public func DCHECK(_ value: Bool, file: String = #file, function: String = #function, line: Int = #line) {
    Objc_DCHECK(value, file, function, Int32(line))
}
